encoders/numeric/CompressionVaeEncoder/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM jinaai/jina
+
+# setup the workspace
+COPY . /workspace
+WORKDIR /workspace
+
+RUN python3 -m pip install --upgrade pip
```

Comment:
Why id this needed?

Reply:
It's not mandatory, I added that cmd just to avoid the `WARNING: You are using pip version 20.0.2; however, version 20.2.3 is available.`

Reply:
If want to do this pleade merge it with another command so that we do not create an extra docker layer